qlty-config/src/config/exclude.rs

```diff
@@ -0,0 +1,106 @@
+use crate::config::issue_transformer::IssueTransformer;
+use globset::{Glob, GlobSet, GlobSetBuilder};
+use qlty_types::analysis::v1::Issue;
+use schemars::JsonSchema;
+use serde::{Deserialize, Deserializer, Serialize};
+use std::sync::RwLock;
+
+fn ensure_non_empty_vec<'de, D, T>(deserializer: D) -> Result<Vec<T>, D::Error>
+where
+    D: Deserializer<'de>,
+    T: Deserialize<'de>,
+{
+    let vec = Vec::<T>::deserialize(deserializer)?;
+    if vec.is_empty() {
+        Err(serde::de::Error::custom("Vector cannot be empty"))
```

Comment:
If this error triggers, will it indicate the field name?

Reply:
Example: 
<img width="851" alt="image" src="https://github.com/user-attachments/assets/ad26e51c-e040-4796-bc4e-2a1e3cb0c527" />

Reply:
Hmm, I think this screenshot is a different case right? (missing `plugins` field is caught by Serde, but this special error is for `[]`)